hudi-hadoop-common/src/main/java/org/apache/hudi/storage/hadoop/HadoopStorageConfiguration.java

```diff
@@ -38,7 +38,11 @@ public class HadoopStorageConfiguration extends StorageConfiguration<Configurati
   private transient Configuration configuration;
 
   public HadoopStorageConfiguration() {
-    this(new Configuration());
+    this(true);
+  }
+
+  public HadoopStorageConfiguration(Boolean loadDefaults) {
```

Comment:
Is this need for production code?  `HadoopStorageConfiguration(Configuration configuration)` should suffice.

Reply:
I don't see any usages where loadDefaults is false. But I wouldn't have done this for no reason. Need to keep looking why

Reply:
Found it. getDefaultStorageConf sets it to false

Reply:
keep as is, but create followup ticket to reduce number of constructors

Reply:
https://issues.apache.org/jira/browse/HUDI-7732
https://issues.apache.org/jira/browse/HUDI-7733